e2e/run_e2e.sh

```diff
@@ -51,3 +51,23 @@ else
     echo "clusterpermission-sample clusterrolebinding not found"
     exit 1
 fi
+
+echo "TEST ClusterPermission with existing roles"
+kubectl config use-context kind-hub
+kubectl apply -f config/samples/clusterpermission_existing_roles.yaml -n cluster1
+sleep 10
+work_kubectl_command=$(kubectl -n cluster1 get clusterpermission clusterpermission-existing-role-sample -o yaml | grep kubectl | grep ManifestWork)
+if $work_kubectl_command; then
+    echo "ManifestWork found"
+else
+    echo "ManifestWork not found"
+    exit 1
+fi
+
+kubectl config use-context kind-cluster1
```

Comment:
@mikeshng One thing I don't get is why the clusterpermission rolebindings are not created on cluster1. ie. for  your test above you found the role and clusterrole on the hub context but shouldn't they be created on cluster1?
<img width="1026" alt="image" src="https://github.com/user-attachments/assets/b74d78ab-b13e-4a8f-b018-84eda35072c4" />

Reply:
The new e2e deploy script that the foundation folks use combine both hub and cluster1 into the same cluster. So think of cluster1 as a local-cluster but still a managed cluster.